types/web3/web3-tests.ts

```diff
@@ -77,3 +77,4 @@ const weiStr: string = web3.utils.toWei("100", "gwei");
 const weiBn: BigNumber = web3.utils.toWei(web3.utils.toBN("1"));
 const rndHex: string = Web3.utils.randomHex(10);
 const sha3: string = web3.utils.soliditySha3(0, 1, "abc");
+const isHex: boolean = web3.utils.isHexStrict("0xc1912");
```

Comment:
please add a test case for the `number` argument type.

Reply:
Ok - added.